plugins/rest/rest.go

```diff
@@ -32,6 +32,11 @@ const (
 	grantTypeJwtBearer         = "jwt_bearer"
 )
 
+var maskedHeaderKeys = map[string]bool{
+	"Authorization":        false,
+	"X-Amz-Security-Token": false,
+}
+
```

Comment:
Nit: Since we don't use the value, we could do `map[string]struct{}`

Reply:
TIL that a `struct` can have a [zero byte](https://go.dev/src/go/types/sizes.go) memory allocation.  Thanks for pointing this out.  I'll make this change, and then I've got some of my own code to go back and refactor 👍